README.md

```diff
@@ -194,6 +194,24 @@ jobs:
             https://cdn.jsdelivr.net/hex
 ```
 
+### OTP Architecture
+
+On Windows you can specify the OTP architecture to install.
+
+```yaml
+# create this in .github/workflows/ci.yml
+on: push
+
+jobs:
+  test:
+    runs-on: windows-latest
+    steps:
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: '26'
+          otp-architecture: '32'
```

Comment:
sorry I haven't responded sooner. Why is this `32` as opposed to `x86`/`x86_64`? I believe the latter would be more future proof as there is slowly arm64-windows support being added: https://github.com/erlang/otp/pull/8991/files.

That being said it is fine to wait for arm builds to show up and we could deprecate 32/64 in favor of x86/x86_64 (note: you're gonna be able to run x86_64/x86_64 on arm64 windows but not the other way around) but I don't see a downside in doing it right now.

Reply:
Hey @wojtekmach , no release has been cut, if we need to we can revert, but also sounds like it's a future proof issue.